src/Domain/Creators/Interval.hpp

```diff
@@ -76,51 +76,61 @@ class Interval : public DomainCreator<1> {
   };
   template <typename BoundaryConditionsBase>
   struct UpperBoundaryCondition {
-    static std::string name() noexcept { return "UpperBoundary"; }
+    static std::string name() noexcept { return "UpperBoundaryCondition"; }
     static constexpr Options::String help =
         "Options for the boundary condition applied at the upper boundary.";
     using type = std::unique_ptr<BoundaryConditionsBase>;
-    using group = BoundaryConditions;
   };
   template <typename BoundaryConditionsBase>
   struct LowerBoundaryCondition {
-    static std::string name() noexcept { return "LowerBoundary"; }
+    static std::string name() noexcept { return "LowerBoundaryCondition"; }
     static constexpr Options::String help =
         "Options for the boundary condition applied at the lower boundary.";
     using type = std::unique_ptr<BoundaryConditionsBase>;
-    using group = BoundaryConditions;
```

Comment:
Why did you remove this group? I liked the `BoundaryConditions:` section in the input file.

Reply:
I had switched to using `Alternatives` at an intermediate stage. I agree, it's nice to have the group. I'll restore that. Thanks!